product_dimension_compute_volume/__init__.py

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
```

Comment:
@carmenbianca Could you adapt your headers to OCA conventions ?

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1

Thanks